src/Control/Email/Email.php

```diff
@@ -2,236 +2,181 @@
 
 namespace SilverStripe\Control\Email;
 
-use DateTime;
-use RuntimeException;
+use Symfony\Component\Mime\Email as SymfonyEmail;
+
 use Egulias\EmailValidator\EmailValidator;
 use Egulias\EmailValidator\Validation\RFCValidation;
+use RuntimeException;
 use SilverStripe\Control\Director;
-use SilverStripe\Control\HTTP;
-use SilverStripe\Core\Convert;
+use SilverStripe\Core\Config\Configurable;
 use SilverStripe\Core\Environment;
+use SilverStripe\Core\Extensible;
+use SilverStripe\Core\Injector\Injectable;
 use SilverStripe\Core\Injector\Injector;
-use SilverStripe\ORM\FieldType\DBDatetime;
 use SilverStripe\ORM\FieldType\DBField;
-use SilverStripe\ORM\FieldType\DBHTMLText;
 use SilverStripe\View\Requirements;
 use SilverStripe\View\SSViewer;
 use SilverStripe\View\ThemeResourceLoader;
 use SilverStripe\View\ViewableData;
-use Swift_Message;
-use Swift_Mime_SimpleMessage;
-use Swift_MimePart;
-
-/**
- * Class to support sending emails.
- */
-class Email extends ViewableData
+use Symfony\Component\Mailer\MailerInterface;
+use Symfony\Component\Mime\Address;
+use Symfony\Component\Mime\Part\AbstractPart;
+
+class Email extends SymfonyEmail
 {
-    /**
-     * @var array
-     * @config
-     */
-    private static $send_all_emails_to = [];
+    use Configurable;
+    use Extensible;
+    use Injectable;
 
-    /**
-     * @var array
-     * @config
-     */
-    private static $cc_all_emails_to = [];
+    private static string|array $send_all_emails_to = [];
 
-    /**
-     * @var array
-     * @config
-     */
-    private static $bcc_all_emails_to = [];
+    private static string|array $cc_all_emails_to = [];
 
-    /**
-     * @var array
-     * @config
-     */
-    private static $send_all_emails_from = [];
+    private static string|array $bcc_all_emails_to = [];
+
+    private static string|array $send_all_emails_from = [];
 
     /**
+     * The default administrator email address or array of [email => name]
```

Comment:
```suggestion
     * The default "from" email address or array of [email => name], or the email address as a string
```
"administrator email address" doesn't really describe what this is used for. Maybe we should even change the name of this to "default_from_email" or similar.

Reply:
Updated